OOVPA.h

```diff
@@ -139,8 +145,8 @@ typedef struct _OOVPATable
 
 // http://en.cppreference.com/w/cpp/iterator/size
 //#include <iterator>
-//#define OOVPA_TABLE_COUNT(x) std::size(x)
-#define OOVPA_TABLE_COUNT(x) (sizeof(x)/sizeof(x[0]))
+//#define XBSDB_ARRAY_SIZE(x) std::size(x)
+#define XBSDB_ARRAY_SIZE(x) (sizeof(x)/sizeof(x[0]))
```

Comment:
If OOVPA.h is only used internally, I'd suggest removing this prefix (and make it `ARRAY_SIZE` - which is a common name for this).
If it's also used externally, this should be changed, and the macro should probably be removed (it's not used within this file, but is very generic).

Reply:
`ARRAY_SIZE` from my understanding aren't included in C standard files nor is included with every compilers. Part of the reason I used the prefix is to avoid macro redefinition plus prevent overridden by some bad macro.

Reply:
As macros are only part of that specific compilation unit, you don't have to worry about a prefix.

Worst case is we'll some day have to add `#undef ARRAY_SIZE` or `#ifndef ARRAY_SIZE` - both extremely unlikely as such macros are only used in projects, and not in headers which are exposed to others.